wiki/markdown/curl-examples.md

```diff
@@ -172,21 +291,64 @@ Note: It is not possible to add the `backendurl` parameter as a JSON parameter.
+The below command would result in a json response of `{"status":true}` if this functionality is enabled.
+
+    curl -X GET -H "Content-type: application/json" localhost:8080/rules/rule-check/testRulePageEnabled
+
+
+Example curl commands to these endpoints [can be found here](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/Running-rules-on-objects.md)
+
+## <a id="subscriptions" />/subscriptions
+
+<table>
+    <tr>
+        <th>Context Path</th>
+        <th>Type</th>
+        <th>Explanation</th>
+    </tr>
+    <tr>
+        <td>/subscriptions</td>
+        <td>GET</td>
+        <td>Fetches all available subscriptions from the back-end</td>
+    </tr>
+    <tr>
+        <td>/subscriptions</td>
+        <td>POST</td>
+        <td>Request to add one (or more) new subscription(s) included in JSON format</td>
+    </tr>
+    <tr>
+        <td>/subscriptions</td>
+        <td>PUT</td>
+        <td>Request to update one (or more) subscription object(s) included in JSON format</td>
+    </tr>
+    <tr>
+        <td>/subscriptions/*subscription name*</td>
+        <td>GET</td>
+        <td>Fetches information about a single (or multiple) subscription(s) from the back-end</td>
+    </tr>
+    <tr>
+        <td>/subscriptions/*subscription name*</td>
+        <td>DELETE</td>
+        <td>Deletes a single (or multiple) subscription(s) from the back-end</td>
+    </tr>
+</table>
+
 
-The `/subscriptions` endpoint is one that may be used the most, we may call that endpoint using three methods `GET`, `POST` and `PUT`.
+The `/subscriptions` endpoint can be called with `GET`, `POST`, `PUT` and `DELETE`.
+More information, and examples, on the `/subscriptions` API can be found [here](https://github.com/eiffel-community/eiffel-intelligence/tree/master/wiki/markdown/Subscription-API.md).
 
 A `POST` request with subscriptions in a file may look as the following example.
 
     curl -X POST -d @file_containing_list_of_json_objects -H "Content-Type: application/json" \
         http://localhost:8080/subscriptions?backendurl="http://127.0.0.1:8090/"
 
 Here is an example using this endpoint and the result it gives in case we have the templated subscriptions added.
+The `backendurl` parameters is passed in to use a specified instance instead of the default back-end instance.
 
     curl -X GET http://localhost:8080/subscriptions?backendurl="http://127.0.0.1:8090/"
 
-The back-end used is running on localhost and port 8080, we redirect the request to 127.0.0.1 and port 8090 as requested in the query parameters and the result is as follows
+The back-end used is running on localhost and port 8080. EI front-end redirects the request to 127.0.0.1 and port 8090 as requested in the query parameters and the result is as follows
```

Comment:
"The back-end"? Is it EI Backend or EI-Frontend webserver you mean? Use "EI-Frotend webserver" or something instead to make it more clear. Often when backend is mentioned in different topics/forums, then it often refers to EI-Backend and EI-Backends webserver.
And "EI front-end redirects" in a web world is that a webserver response back with a new url to use where the actual web service has moved to, which is incorrect. What actual happening in EI-F webserver is that all request is forwarded to the specified EI Backend instance. This functionality/service we named to "bridge". So I think "forwarding requests" or "bridging requests" is better word to use.

Reply:
Fixed.

Reply:
I think redirect in a  web world is not about forwarding a request, it gives a response back to the web browser that this is the new url to use. Read more about it here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections
I think the word "forward the request" or "bridge the request" is more correct.

Reply:
Ah, you have update that already :)